### PR TITLE
MaSuRCA for foss 2016a

### PR DESCRIPTION
**Pull request by ysagon**

### CODE REVIEW

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+buildopts="install-special"
+start_dir="global-1"
```
Comment (boegel) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb]: please move this down, below `dependencies`

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+#need a temporary url to download it. Do it manually.
+source_urls = ['http://www.genome.umd.edu/masurca_form.html']
```
Comment (boegel) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb]: don't specify this via `source_urls`, but just as a comment, there's no point in letting EasyBuild attempt a download

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+dependencies = [
+    ('libreadline', '6.3'),
+    ('Tcl', '8.6.5'),
+    ('Boost', '1.61.0'),
+    ('zlib', '1.2.8'),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["bin"],
```
Comment (boegel) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb]: Maybe make this a bit more specific?

```python
sanity_check_paths = {
    'files': ['bin/masurca'],
    'dirs': ['include', 'lib'],
}

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+sources = ['%(name)s-%(version)s.tar.gz']
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('Tcl', '8.6.5'),
+    ('Boost', '1.61.0'),
+    ('zlib', '1.2.8'),
+]
+
+toolchain = {'name': 'foss', 'version': '2016a'}
```
Comment (boegel) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb]: Sorry if I wasn't clear, this should stay up (above `sources`) style-wise...

We hope to start automatic style checks soon, including order of parameters.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3b50fda2bf416ac8057e7130cd4dd093 for a full test report.

File: easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb (hunk 4 above)
Comment (ysagon) [inline easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.2.2-foss-2016a.eb]: Is there a document where the style required to write eb files is specified?

**Comment by boegel** [comment]
@ysagon We have this: http://easybuild.readthedocs.io/en/latest/Contributing.html#code-style-review, but it mostly applies to Python *code*, i.e. it doesn't formalise desired order of easyconfig parameters.

There's been some work on making `eb --check-style` also check ordering, but it hasn't been finished yet, see https://github.com/hpcugent/easybuild-framework/pull/2165.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ffc52dde1801aa7ba7185baf7865c986 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ysagon!

**Pull request merged**